mayhem/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google Inc.
```

Comment:
Not sure if we need the copyright at the top here. Also, can you add a package stage?

Reply:
I'm not sure either, on one hand this is very trivial code, on the other hand, I did copy and paste it. I can add a package stage once the other questions are addressed, as I have a few repos with this same exact line of questioning, namely 
- package stage
- copyright
- build script